.github/workflows/ci.yml

```diff
@@ -30,4 +25,4 @@ jobs:
       - name: Run the tests
         env:
           DISTRO: ${{ matrix.distro }}
-        run: make PYTEST_ADDOPTS="${PYTEST_ADDOPTS}" test-docker
```

Comment:
Why drop the `PYTEST_ADDOPTS` part? That should still be relevant to running in podman, is it not? This way, functional tests are run when running in cronjob.

Reply:
You are right, it's not necessary